runtime/list_test.go

```diff
@@ -353,6 +353,39 @@ func TestListIteratorIter(t *testing.T) {
 	}
 }
 
+func TestListPop(t *testing.T) {
+	fun := newBuiltinFunction("TestListPop", func(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
```

Comment:
I would get rid of the kwargs here and have the function return (pop result, list). Something like:

```
pop := mustNotRaise(GetAttr(NewRootFrame(), ListType.ToObject(), NewStr("pop"), nil))
fun := wrapFuncForTest(func(f *Frame, l *List, args ...*Object) (*Tuple, *BaseException) {
  result, raised := pop.Call(f, wrapArgs(l, args...), nil)
  if raised != nil {
    return nil, raised
  }
  return newTestTuple(result, l), nil
})
```

Sorry for wrapFuncForTest() being poorly documented. It's pretty smart about checking that arguments are of the right type and variadic arguments and such so it's a nice way to avoid having to do a lot of validation inside a `func(*Frame, Args, KWArgs)`.